apps/documentation/src/routes/documentation/tutorial/api-fetching.mdx

```diff
@@ -1,19 +1,19 @@
-import { Head } from 'tuono'
+import MetaTags from '../../../components/meta-tags'
 
-<Head>
-  <title>Tutorial - API fetching</title>
-  <link
-    rel="canonical"
-    href="https://tuono.dev/documentation/tutorial/api-fetching"
-  />
-</Head>
+<MetaTags
+  title="Tuono - API fetching"
+  canonical="https://tuono.dev/documentation/tutorial/api-fetching"
+  description="Learn how to fetch remote APIs on the backend"
+/>
 
 import Breadcrumbs, { Element } from '../../../components/breadcrumbs'
 
-<Breadcrumbs>
-  <Element label="Tutorial" href="/documentation/tutorial" />
-  <Element label="API fetching" />
-</Breadcrumbs>
+<Breadcrumbs
```

Comment:
**[LanguageTool]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Possible typo: you repeated a word (ENGLISH_WORD_REPEAT_RULE)
Suggestions: `Breadcrumbs`
Rule: https://community.languagetool.org/rule/show/ENGLISH_WORD_REPEAT_RULE?lang=en-US
Category: MISC
<!-- __reviewdog__:ChBjMjFiY2Y4MTJkODM0MDc2EgxMYW5ndWFnZVRvb2w= -->